datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -100,7 +100,9 @@ export class DatasetEntity implements Entity<Dataset> {
                     component: ViewDefinitionTab,
                     display: {
                         visible: (_, dataset: GetDatasetQuery) =>
-                            (dataset?.dataset?.subTypes?.typeNames?.includes(SUBTYPES.VIEW) && true) || false,
+                            dataset?.dataset?.subTypes?.typeNames
+                                ?.map((t) => t.toLocaleLowerCase())
+                                .includes(SUBTYPES.VIEW.toLocaleLowerCase()) || false,
```

Comment:
Is this for backwards compatibility? Seems simpler to change `SUBTYPES.VIEW` to `View`

Reply:
yeah for backwards compatibility! and just defensive in case it ever changes again, might as well compare them as a standard